utils/findutils.xml

```diff
@@ -0,0 +1,132 @@
+       directories  that  are  writable  by  other  users), you
+       should read the &quot;Security Considerations&quot; chapter of the
+       findutils  documentation,  which is called Finding Files
+       and comes with findutils.   That document also  includes
+       a  lot more detail and discussion than this manual page,
+       so you may find it a more useful source of  information.</description>
+  </entry-point>
+  <entry-point binary-name="locate" command="locate">
+    <needs-terminal/>
+    <name xml:lang="en">locate</name>
+    <summary xml:lang="en">locate - list files in databases that match a pattern</summary>
+    <description xml:lang="en">For each given pattern,  locate  searches  one  or  more
+       databases of file names and displays the file names that
+       contain the pattern.  Patterns can  contain  shell-style
+       metacharacters:  `*', `?', and `[]'.  The metacharacters
+       do not treat `/' or `.'  specially.  Therefore,  a  pat-
+       tern  `foo*bar'  can  match  a  file  name that contains
+       `foo3/bar', and a pattern `*duck*' can match a file name
+       that  contains  `lake/.ducky'.   Patterns  that  contain
+       metacharacters should be quoted  to  protect  them  from
+       expansion by the shell.</description>
+  </entry-point>
+  <entry-point binary-name="xargs" command="xargs">
+    <needs-terminal/>
+    <name xml:lang="en">xargs</name>
+    <summary xml:lang="en">xargs  -  build  and execute command lines</summary>
+    <description xml:lang="en">xargs  reads items from the standard input, delimited by
+       blanks (which can be protected  with  double  or  single
+       quotes  or  a  backslash)  or newlines, and executes the
+       command (default is /bin/echo) one or  more  times  with
+       any  initial-arguments followed by items read from stan-
+       dard input.  Blank  lines  on  the  standard  input  are
+       ignored.</description>
+  </entry-point>
+  <entry-point binary-name="updatedb" command="updatedb">
+    <needs-terminal/>
+    <name xml:lang="en">updatedb</name>
+    <summary xml:lang="en">updatedb - update a file name database</summary>
+    <description xml:lang="en">The file name databases contain lists of files that were
+       in particular directory trees when  the  databases  were
+       last  updated.  The file name of the default database is
+       determined when locate and updatedb are  configured  and
+       installed.   The  frequency with which the databases are
+       updated and  the  directories  for  which  they  contain
+       entries  depend  on  how often updatedb is run, and with
+       which arguments.
+</description>
+  <entry-point command="updatedb" binary-name="updatedbbat">
+    <needs-terminal />
+    <name xml:lang="en">updatedb bat</name>
+    <summary xml:lang="en">This is a pale imitation of the original `updatedb.sh'</summary>
+  </entry-point>
+  </entry-point>
```

Comment:
This `</entry-point>` needs to be moved up to line 126.

Reply:
moved